LearningMissionSimulation/PlaygroundEdvard.cs

```diff
@@ -104,65 +54,100 @@ public void CreateAccounts(int accountCount)
 
                 Console.WriteLine('\n');
             }
+
+            ReportSummary("Account", accountCount);
+            ReportFooter("Create Account");
         }
 
         public void ActivateAccounts()
         {
-            Console.WriteLine("-----Unfair coordinator started activating accounts-----\n");
+            ReportHeader("Activate account");
 
             while (pendingAccountStack.Count > 0)
             {
                 pendingAccountStack.Pop().Status = Status.Active;
```

Comment:
Where is the item report inside the loop? You have to split pendingAccountStack.Pop().Status = Status.Active;
Account pendingAccount = pendingAccountStack.Pop();
pendingAccount.Status = Status.Active;
and then call reportItem().